src/App.jsx

```diff
@@ -1,27 +1,72 @@
+import { setUtahHeaderSettings } from '@utahdts/utah-design-system-header';
+import { useEffect } from 'react';
 import { useRemoteConfigString } from 'reactfire';
-import './App.css';
-import { fieldNames } from './config';
+import MapComponent from './components/Map.jsx';
+import SearchWizard from './components/search-wizard/Wizard.jsx';
 
 function App() {
   const queryLayersConfig = useRemoteConfigString('queryLayers');
+  const linksConfig = useRemoteConfigString('links');
+
+  useEffect(() => {
+    if (linksConfig.status !== 'success') {
+      return;
+    }
+
+    const links = JSON.parse(linksConfig.data);
+    setUtahHeaderSettings({
+      // this prop's implementation has not been released yet
+      // domLocationTarget: {
+      //   element: headerRef.current,
+      // },
+      title: 'Utah Environmental Interactive Map',
+      titleURL: null,
+      logo: '<img src="/deq_logo.png" alt="DEQ Logo" />',
+      utahId: false,
+      actionItems: [
+        {
+          actionPopupMenu: {
+            menuItems: [
+              {
+                actionUrl: {
+                  url: links['2'].url,
+                  openInNewTab: true,
+                },
+                title: links['2'].description,
+              },
+              {
+                // TODO: add a modal for this
+                actionFunction: () => window.alert('not implemented yet!'),
+                title: 'Disclaimer',
+              },
+            ],
+            title: 'Links',
+          },
+          className: 'icon-waffle',
+          showTitle: false,
+          title: 'Links',
+          icon: '<span class="utds-icon-before-waffle" aria-hidden="true" />',
+        },
+      ],
+    });
+  }, [linksConfig]);
 
   if (queryLayersConfig.status === 'loading') {
     console.log('loading remote config');
 
-    return <span>getting configs...</span>;
+    return null;
   }
 
   const queryLayers = JSON.parse(queryLayersConfig.data);
-  console.log('queryLayers', queryLayers);
 
   return (
-    <div className="App">
-      <h1>Query Layers</h1>
-      {queryLayers.map((queryLayer) => (
-        <div key={queryLayer[fieldNames.queryLayers.name]}>
-          {queryLayer[fieldNames.queryLayers.name]}
-        </div>
-      ))}
+    <div className="h-full w-full flex">
+      <div className="flex-1 border-r border-gray-300">
+        <MapComponent />
+      </div>
+      <div className="w-80 overflow-y-auto">
```

Comment:
**suggestion**: consider using non-modal Dialog as the underlying primitive for the drawer https://github.com/radix-ui/primitives/issues/986

**suggestion**: consider the layouts from the suggested  [Utah Design System Mapping.pdf](https://github.com/agrc/deq-enviro/files/11335044/Utah.Design.System.Mapping.pdf). Also think about the small screen layout with responsive design.

https://tailwindcss.com/docs/responsive-design

Reply:
I'm not sure how much time to dedicate to mobile layouts for this app. @nathankota Do you see it as a priority? 

Here are some analytics for the last six months or so...

<img width="838" alt="image" src="https://user-images.githubusercontent.com/1326248/234913598-1b4943d8-d955-4829-8a15-d168be19af70.png">

Reply:
>consider using non-modal Dialog as the underlying primitive for the drawer

I'm not sure what that gets us since this isn't a drawer that slides in and out. Or perhaps you are suggesting that it become a drawer?

Reply:
The start to a basic mobile layout: e1d18d06e3ab1e7e4e7a862a0b5a7ce7496f7482